source/includes/_sequences.md

```diff
@@ -10,38 +15,54 @@ What is a sequence?
     "name": "Some series",
     "prefix": "XX",
     "padding": 7,
-    "suffix": "Y",
-    "ownerId": "db0db654-1694-4535-ae91-4506a206e4c2"
+    "suffix": "Y"
 }
 ```
 
-Information about the code object
+The object has three properties to create the terms of its sequence, `prefix`,
+`padding` and `suffix`. All the terms will share the same `prefix` and
+`suffix`, `padding` tells how many digits are set (at most) between those fixed
+values. Those digits represent a numerical value that always starts with one.
+Independently of the actual amount of digits of that number, the `padding`
+ensures the size of that portion of the entry's value, filling the gap with
+leading zeros.
 
 ## The Entry object
 
 ```json
 {
     "id": "550d89db-2b53-476c-a011-f796e196b053",
-    "codeId": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
+    "code_id": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
     "idx": 186,
-    "value": "XX0000186Y"
+    "value": "XX0000186Y",
+    "prev_id": "..."
 }
 ```
 
-Information about the entry object
+Each entry has a value built from the code properties, and it's linked to other entry. Except for the first one. Similar to a singly linked list. The following object would be the next entry for the previous object.
+
+```json
+{
+    "id": "2a8ece9f-7590-4e8b-a4f5-82d1d0ee16f6",
+    "code_id": "d4cfb49f-035b-46f0-b7b4-ec733aee2d00",
+    "idx": 187,
+    "value": "XX0000187Y",
+    "prev_id": "550d89db-2b53-476c-a011-f796e196b053"
+}
+```
 
 ## Create a code
 
 ```shell
-curl "https://api.invopop.dev/sequence/:owner_id/code" \
+curl --request POST "https://api.invopop.dev/sequence/:owner_id/code" \
   -H "Authorization: Bearer your.api.key"
   -H "Content-Type: application/json"
-  -d '{ \
-        "id":"175ec7e5-8c14-46fd-90ff-ccb37d71550c", \
-        "name":"Some series", \
-        "prefix": "XX", \
-        "suffix": "Y", \
-        "padding": 7 \
+  -d '{
+        "id":"175ec7e5-8c14-46fd-90ff-ccb37d71550c",
```

Comment:
The idea is that the examples could be directly copied from here, so we shouldn't remove the backslashes from the end of the lines.

Reply:
The curl request works without the backslashes in the data object, you are opening a string with a quote and waits for a closing one. Although, the headers are missing their backslash.